### DISCUSSION TITLE
Some identifiers in Javascript/Typescript have (wrong?) colors

### DISCUSSION THREAD

**Discussion by fynnfluegge** [catppuccin/nvim#522] [Q&A] [answered]
Hi,
I really like the catppuccin theme but with javascript/typescript there are some keywords, e.g.  from `global`, `window` objects  or the `this` keyword red colored as in the following screenshot:
![Screenshot 2023-06-24 at 18 46 57](https://github.com/catppuccin/nvim/assets/16321871/abcde80f-c3be-4411-8061-5d5f1efe0dce)
The red color is really annoying, I tried to adjust any kind of groups in my custom configuration which has non effect. In the [syntax.lua](https://github.com/catppuccin/nvim/blob/main/lua/catppuccin/groups/syntax.lua) there is only the neovim `Error` red colored. Overriding that color will also override the previously mentioned keywords but this is not a proper solution. 
The errors should remain red. Any ideas how I can tweak the theme for my purposes? I discover that behaviour only in the catppuccin theme. In my other themes these keywords have the same color as `Identifier` syntax group.

**Comment by mrtnvgr** [comment, marked as answer]
You can use `:Inspect` command to view highlight group at the cursor position.
